src/breeze/actions/emails/show.cr

```diff
@@ -0,0 +1,38 @@
+abstract class Breeze::Emails::Show < BreezeAction
```

Comment:
You have to create your own action class that inherits from this for it to work. I don't think it's horrible, but will just need a very good way to hold the user's hand on how to do this.

Reply:
@jwoertink Maybe we make a `breeze.email_preview.init` task kind of like I did with Pundit here?

https://github.com/stephendolan/pundit/blob/main/tasks/initialize.cr

It could be a nice way to offload the documentation and a good standard to move forward with for this kind of setup step.

Reply:
Oh, you mean the task would create this file for the user so they don't have to set it up? That might be ok. We will just need to figure out how we tell them to run that. Docs, obviously, but if there was some step where we can print that.... Maybe the install just does it? Or the install prints out something like "If you want to view emails, run ...." 🤔

Reply:
The alternative would be to provide a way to register emails for the action to display. Like providing a name and a proc that would return the email and it would get stored in a hash.

Rails has you make these preview classes https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

We could do the same where the abstract class registers itself when it is inherited.